library.sh

```diff
@@ -679,14 +706,16 @@ function go_update_deps() {
   update_licenses third_party/VENDOR-LICENSE "./..."
 
   group "Removing broken symlinks"
-  remove_broken_symlinks ./vendor
+  remove_broken_symlinks vendor
```

Comment:
is there a reason for removing the `./` prefix here? usually in shell scripts it is conventional to leave that to indicate this is a local directory

Reply:
No big reason, just a simplification.

The `./` prefix isn't really required. It doesn't save us from anything, but might be cumbersome if tried to concatenate paths, for example.

Reply:
Unless we are doing the concat i would keep the `./` just for convention reasons, but just my 2cents not a blocker